modules/multiprocessExample/main.py

```diff
@@ -0,0 +1,250 @@
+import multiprocessing as mp
+# import pathos.multiprocessing as pm
+import time
+
+import counter
+import printer
+
+
+# If the queue is empty then the exit was not requested
+def exitRequested(requestQueue):
+    return not requestQueue.empty()
+
+
+def counterWorker(counterClass, pause, exitRequest, pipelineOut):
```

Comment:
Instantiate class objects inside the worker function, pass constructor arguments here instead.